package.json

```diff
@@ -62,6 +60,7 @@
     "react-router-dom": "==6.0.0-beta.4"
   },
   "dependencies": {
+    "@gliff-ai/style": "^3.0.0",
```

Comment:
I thought we kept this as a peer+dev dependency, so we don't end up bringing multiple versions into DOMINATE?

Reply:
Yea, it definitely should be